integration/distinct_compat_test.go

```diff
@@ -156,10 +155,6 @@ func TestDistinctCompat(t *testing.T) {
 			field:  "v.0.foo",
 			filter: bson.D{},
 		},
-		"QueryNullType": {
-			field:  "v",
-			filter: types.Null,
```

Comment:
@b1ron that should be `nil`, not `types.Null`. MongoDB Go driver knows nothing about FerretDB code types.

Reply:
P. S. And please don't work on your day off

Reply:
OK I will stop now :+1: